### PR TITLE
India carbon factors

### PR DESCRIPTION
**Pull request by tomhebunn**
feat: added India dataset from Raj Patel in the Mumbai office

### CODE REVIEW

File: src/store/utilities/material-carbon-factors.ts [hunk 1]
```diff
@@ -12,317 +12,598 @@ export interface Material {
...
+  "Insulation",
+  "Plasterboard",
+  "Plastic",
+  "Steel",
+  "Stone",
+  "Timber"
+
+]
+
 export interface UKMaterialCarbonFactors {
```
Comment (Exilliar) [inline src/store/utilities/material-carbon-factors.ts:40]: This interface could be renamed to `MaterialCarbonFactors`, and then the current `MaterialCarbonFactors` could be renamed to be `AllMaterialCarbonFactors` (or something like that)

File: src/store/utilities/material-carbon-factors.ts [hunk 2]
```diff
@@ -12,317 +12,598 @@ export interface Material {
...
 export interface MaterialCarbonFactors {
-  UK: UKMaterialCarbonFactors;
-  Australia: any;
-  EU: any;
-  "East Asia": any;
+  India: any;
+  UK: any;
+  // Australia: any;
+  // EU: any;
+  // EastAsia: any;
```
[the remark this answers is not in the document]
Comment (tomhebunn) [reply src/store/utilities/material-carbon-factors.ts]: done

File: src/store/utilities/material-carbon-factors.ts (hunk 1 above)
Comment (tomhebunn) [reply src/store/utilities/material-carbon-factors.ts]: done

File: src/store/utilities/material-carbon-factors.ts [hunk 3]
```diff
@@ -12,317 +12,598 @@ export interface Material {
 }
 
 export interface MaterialCarbonFactors {
-  UK: UKMaterialCarbonFactors;
-  Australia: any;
-  EU: any;
-  "East Asia": any;
+  India: any;
+  UK: any;
```
[the remark this answers is not in the document]
Comment (tomhebunn) [reply src/store/utilities/material-carbon-factors.ts:16]: done

**Review by Exilliar — APPROVED** [review]
all looks like it's working, good to merge

**Pull request merged**